content-review/impl/vericite/src/main/java/org/sakaiproject/contentreview/vericite/client/ApiClient.java

```diff
@@ -1,658 +1,1324 @@
-      delimiter = "|";
-    }
-
-    StringBuilder sb = new StringBuilder() ;
-    for (Object item : valueCollection) {
-      sb.append(delimiter);
-      sb.append(parameterToString(item));
-    }
-
-    params.add(new Pair(name, sb.substring(1)));
-
-    return params;
-  }
-
-  /**
-   * Check if the given MIME is a JSON MIME.
-   * JSON MIME examples:
-   *   application/json
-   *   application/json; charset=UTF8
-   *   APPLICATION/JSON
-   */
-  public boolean isJsonMime(String mime) {
-    return mime != null && mime.matches("(?i)application\\/json(;.*)?");
-  }
-
-  /**
-   * Select the Accept header's value from the given accepts array:
-   *   if JSON exists in the given array, use it;
-   *   otherwise use all of them (joining into a string)
-   *
-   * @param accepts The accepts array to select from
-   * @return The Accept header to use. If the given array is empty,
-   *   null will be returned (not to set the Accept header explicitly).
-   */
-  public String selectHeaderAccept(String[] accepts) {
-    if (accepts.length == 0) {
-      return null;
-    }
-    for (String accept : accepts) {
-      if (isJsonMime(accept)) {
-        return accept;
-      }
-    }
-    return StringUtil.join(accepts, ",");
-  }
-
-  /**
-   * Select the Content-Type header's value from the given array:
-   *   if JSON exists in the given array, use it;
-   *   otherwise use the first one of the array.
-   *
-   * @param contentTypes The Content-Type array to select from
-   * @return The Content-Type header to use. If the given array is empty,
-   *   JSON will be used.
-   */
-  public String selectHeaderContentType(String[] contentTypes) {
-    if (contentTypes.length == 0) {
-      return "application/json";
-    }
-    for (String contentType : contentTypes) {
-      if (isJsonMime(contentType)) {
-        return contentType;
-      }
-    }
-    return contentTypes[0];
-  }
-
-  /**
-   * Escape the given string to be used as URL query value.
-   */
-  public String escapeString(String str) {
-    try {
-      return URLEncoder.encode(str, "utf8").replaceAll("\\+", "%20");
-    } catch (UnsupportedEncodingException e) {
-      return str;
-    }
-  }
-
-  /**
-   * Serialize the given Java object into string according the given
-   * Content-Type (only JSON is supported for now).
-   */
-  public Object serialize(Object obj, String contentType, Map<String, Object> formParams) throws ApiException {
-    if (contentType.startsWith("multipart/form-data")) {
-      FormDataMultiPart mp = new FormDataMultiPart();
-      for (Entry<String, Object> param: formParams.entrySet()) {
-        if (param.getValue() instanceof File) {
-          File file = (File) param.getValue();
-          mp.bodyPart(new FileDataBodyPart(param.getKey(), file, MediaType.MULTIPART_FORM_DATA_TYPE));
+    public static final double JAVA_VERSION;
+    public static final boolean IS_ANDROID;
+    public static final int ANDROID_SDK_VERSION;
```

Comment:
This code looks auto generated I would look at removing the Android code as it doesn't apply here.